src/array/cpu/spmm.h

```diff
@@ -19,56 +19,14 @@
 
 #include "spmm_binary_ops.h"
 #if !defined(_WIN32)
-#ifdef USE_AVX
```

Comment:
Why is `USE_AVX` removed?

Reply:
Interesting. Is there a reason it was set up this way? I initially thought `USE_AVX` was added to workaround some build failures.
Right now it seems redundant, it wraps the same code USE_LIBXSMM wraps